savant/converter/yolo.py

```diff
@@ -34,6 +39,50 @@ def __init__(
         self.class_ids = class_ids
         super().__init__()
 
+    def _decode_single_head(
+        self, output: np.ndarray, num_classes: Optional[int]
+    ) -> tuple[np.ndarray, np.ndarray, np.ndarray]:
+        """
+        Returns (bboxes_xywh, confidences, class_ids) from a single output tensor.
+        Handles both [N, 4+C]/[N, 5+C] and transposed [(4+C),N]/[(5+C),N].
+        """
+        # Squeeze batch dim if present
+        if output.ndim == 3 and output.shape[0] == 1:
```

Comment:
Can you explain the case when we receive batch in dims? If this is not possible, the check should be removed.